packages/charts/src/renderers/line/line-accessors.ts

```diff
@@ -37,14 +37,7 @@ export class LineAccessors extends XYAccessors implements ILineAccessors {
     public data: ILineDataAccessors;
     public series: ILineSeriesAccessors;
 
-    constructor(private colorProvider = defaultColorProvider(), private markerProvider = defaultMarkerProvider()) {
-        super();
-
-        this.data.defined = (d: any, i) => d.hasOwnProperty("defined") ? d.defined : true;
-
-        this.series = {
-            color: this.colorProvider ? this.colorProvider.get : undefined,
-            marker: this.markerProvider ? this.markerProvider.get : undefined,
-        };
```

Comment:
It makes more sense to me to leave these here instead of moving them up to the XYRenderer which doesn't draw anything on the chart.